yesod-auth-hashdb.cabal

```diff
@@ -38,7 +38,7 @@ library
                    , yesod-auth              >= 1.4.18     && < 1.7
                    , text                    >= 0.7
                    , yesod-persistent        >= 1.2
-                   , persistent              >= 2.1        && < 2.11
+                   , persistent              >= 2.1        && < 2.12
```

Comment:
As mentioned in the PR comments, we could omit this upper bound entirely, and instead switch to an upper bound on `yesod-persistent`. This would be more consistent with how the package uses `persistent`.

The test suite does use `persistent-sqlite`, and a version bound on there might be appropriate too.